seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/src/test/resources/fake_to_hbase_with_multipletable.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  FakeSource {
+    tables_configs = [
+      {
+          row.num = 5
+          schema {
+            table = "hbase_sink1"
+            fields {
+              name = string
+              age = int
+              c_tinyint = tinyint
+              c_smallint = smallint
+              c_bigint = bigint
+              c_float = float
+              c_double = double
+              c_boolean = boolean
+            }
+          }
+      },
+      {
+          row.num = 6
+          schema {
+            table = "hbase_sink2"
+            fields {
+              name = string
+              age = int
+              c_tinyint = tinyint
+              c_smallint = smallint
+              c_bigint = bigint
+              c_float = float
+              c_double = double
+              c_boolean = boolean
+            }
+          }
+      }
+    ]
+  }
+}
+
+
+sink {
+  Hbase {
+    zookeeper_quorum = "hbase_e2e:2181"
+    table = "seatunnel_test"
```

Comment:
How does it support writing to multiple tables?

Reply:
Please add test cases to write to tables with different structures

Reply:
> How does it support writing to multiple tables?

Maybe I have a problem with my understanding. Is it many-to-many that we're going to end up with ?

Reply:
You decide how to design this function.

e.g. multiple tables as input upstream:
```
source {
  FakeSource {
    tables_configs = [
      {
          schema {
            table = "table-11111"
            fields {
                id = string
                age = int
            }
          }
      },
      {
          schema {
            table = "table-2222"
            fields {
               f1 = boolean
               f2 = boolean
               f3 = tinyint
            }
          }
      }
    ]
  }
}
```

Reply:
> @BruceWong96
> 
> You decide how to design this function.
> 
> e.g. multiple tables as input upstream:
> 
> ```
> source {
>   FakeSource {
>     tables_configs = [
>       {
>           schema {
>             table = "table-11111"
>             fields {
>                 id = string
>                 age = int
>             }
>           }
>       },
>       {
>           schema {
>             table = "table-2222"
>             fields {
>                f1 = boolean
>                f2 = boolean
>                f3 = tinyint
>             }
>           }
>       }
>     ]
>   }
> }
> ```

```
env {
  parallelism = 1
  job.mode = "BATCH"
}


source {
  FakeSource {
    tables_configs = [
      {
          schema {
            table = "table-11111"
            fields {
                id = string
                age = int
            }
          }
      },
      {
          schema {
            table = "table-2222"
            fields {
               f1 = boolean
               f2 = boolean
               f3 = tinyint
            }
          }
      }
    ]
  }
}

sink {
  Hbase {
    zookeeper_quorum = "hadoop001:2181,hadoop002:2181,hadoop003:2181"
    tables_configs = [
      {
        source_table_name = "table-11111"
        table = "hbase_table_1"
        rowkey_column = ["id"]
        family_name {
          id = "cf1"
          age = "cf1"
        }
      },
      {
        source_table_name = "table-2222"
        table = "hbase_table_2"
        rowkey_column = ["f1"]
        family_name {
          f1 = "cf2"
          f2 = "cf2"
          f3 = "cf2"
        }
      }
    ]
  }
}

```
Please check whether this configuration meets the requirements. If so, I will develop according to this configuration.
Thanks.